baseten-performance-client-typescript/baseten-performance-client/.github/workflows/CI.yml

```diff
@@ -0,0 +1,484 @@
+name: CI
+env:
```

Comment:
this is propably auto-generated, but most likley we will need to move it to root from baseten-performance-client-typescript